code/game/objects/items/food/salad.dm

```diff
@@ -101,6 +101,15 @@
 	tastes = list("rice" = 1, "meat" = 1)
 	foodtypes = GRAIN | MEAT
 
+/obj/item/food/salad/risotto
+	name = "risotto"
+	desc = "Proof the Italians mastered every carb."
+	icon_state = "risotto"
```

Comment:
if we took the sprites from polaris, shouldn't we credit them somewhere?

Reply:
theyre on the pr